src/SFA.DAS.AssessorService.Web.Staff/Views/Register/EditOrganisation.cshtml

```diff
@@ -72,6 +72,20 @@ Layout = "_Layout";
                     </label>
                     <input class="form-control" name="LegalName" sfa-validation-for="LegalName" sfa-validationerror-class="form-control-error" asp-for="LegalName" type="text" />
                 </div>
+                <div class="form-group" sfa-validation-for="CompanyNumber" sfa-validationerror-class="form-group-error">
+                    <label for="CompanyNumber">
+                        <span class="form-label-bold">Company number (optional)</span>
```

Comment:
We haven't used the phrase '(optional)' anywhere else in this app, so consider removing it in both places? 
I'll leave that to your discretion, I won't make not approve because of this.